fvm/src/machine/manifest.rs

```diff
@@ -124,13 +140,38 @@ impl Manifest {
             .get(ETHACCOUNT_ACTOR_NAME)
             .context("manifest missing ethaccount actor")?;
 
+        let storagemarket_code = *by_name
+            .get(STORAGE_MARKET_ACTOR_NAME)
+            .context("manifest missing storagemarket actor")?;
+
+        let storagepower_code = *by_name
+            .get(STORAGE_POWER_ACTOR_NAME)
+            .context("manifest missing storagepower actor")?;
+
+        let verifiedregistry_code = *by_name
+            .get(VERIFIED_REGISTRY_ACTOR_NAME)
+            .context("manifest missing verifiedregistry actor")?;
+
+        let datacap_code = *by_name
+            .get(DATA_CAP_ACTOR_NAME)
+            .context("manifest missing datacap actor")?;
+
+        let reward_code = *by_name
+            .get(REWARD_ACTOR_NAME)
+            .context("manifest missing reward actor")?;
+
```

Comment:
We should just store the "by name" mapping in the manifest object, and look these up in the tests. This manifest object is really designed for use by the FVM _machine_ itself and shouldn't explicitly have any understanding of other actors.

Reply:
Eh, we can refactor this part later. The code here is just following the existing conventions.